macos/FluentUI/Colors.swift

```diff
@@ -0,0 +1,304 @@
+			case .warningShade20:
+				return "warningShade20"
+			case .warningShade10:
+				return "warningShade10"
+			case .successPrimary:
+				return "successPrimary"
+			case .successTint40:
+				return "successTint40"
+			case .successTint30:
+				return "successTint30"
+			case .successTint20:
+				return "successTint20"
+			case .successTint10:
+				return "successTint10"
+			case .successShade30:
+				return "successShade30"
+			case .successShade20:
+				return "successShade20"
+			case .successShade10:
+				return "successShade10"
+			case .presenceAvailable:
+				return "presenceAvailable"
+			case .presenceAway:
+				return "presenceAway"
+			case .presenceBlocked:
+				return "presenceBlocked"
+			case .presenceBusy:
+				return "presenceBusy"
+			case .presenceDnd:
+				return "presenceOffline"
+			case .presenceOffline:
+				return "presenceOffline"
+			case .presenceOof:
+				return "presenceOof"
+			case .presenceUnknown:
+				return "presenceUnknown"
+			}
+		}
+	}
+	
+	// MARK: Physical - grays
+	
+	@objc public static let gray950: NSColor = Palette.gray950.color
+	@objc public static let gray900: NSColor = Palette.gray900.color
+	@objc public static let gray800: NSColor = Palette.gray800.color
+	@objc public static let gray700: NSColor = Palette.gray700.color
+	@objc public static let gray600: NSColor = Palette.gray600.color
+	@objc public static let gray500: NSColor = Palette.gray500.color
+	@objc public static let gray400: NSColor = Palette.gray400.color
+	@objc public static let gray300: NSColor = Palette.gray300.color
+	@objc public static let gray200: NSColor = Palette.gray200.color
+	@objc public static let gray100: NSColor = Palette.gray100.color
+	@objc public static let gray50: NSColor = Palette.gray50.color
+	@objc public static let gray25: NSColor = Palette.gray25.color
+	
+	// MARK: Physical - Non-grays
+	
+	@objc public static let error: NSColor = Palette.dangerPrimary.color
+	@objc public static let warning: NSColor = Palette.warningPrimary.color
+	
+	@objc public static var avatarBackgroundColors: [NSColor] = [
+		Palette.cyanBlue10.color,
+		Palette.red10.color,
+		Palette.magenta20.color,
+		Palette.green10.color,
+		Palette.magentaPink10.color,
+		Palette.cyanBlue20.color,
+		Palette.orange20.color,
+		Palette.cyan20.color,
+		Palette.orangeYellow20.color,
+		Palette.red20.color,
+		Palette.blue10.color,
+		Palette.magenta10.color,
+		Palette.gray40.color,
+		Palette.green20.color,
+		Palette.blueMagenta20.color,
+		Palette.pinkRed10.color,
+		Palette.gray30.color,
+		Palette.blueMagenta30.color,
+		Palette.gray20.color,
+		Palette.cyan30.color,
+		Palette.orange30.color
+	]
+	
+	/// Used for hyperlinks
+	@objc public static let communicationBlue: NSColor = Palette.communicationBlue.color
+	
+	@objc public static func color(from palette: Palette) -> NSColor {
+		return palette.color
+	}
+}
+
+/// Make palette enum CaseIterable for unit testing purposes
+extension Colors.Palette: CaseIterable {}
```

Comment:
Can this happen in test code instead of production code?

Reply:
yes

Reply:
actually, we might need a workaround for that.